integrationTests/testProcessorNode.go

```diff
@@ -82,11 +82,11 @@ var TestVmMarshalizer = &marshal.JsonMarshalizer{}
 // TestTxSignMarshalizer represents the marshalizer used in vm communication
 var TestTxSignMarshalizer = &marshal.JsonMarshalizer{}
 
-// TestPubkeyConverter represents a plain public key converter
-var TestPubkeyConverter, _ = pubkeyConverter.NewHexPubkeyConverter(32)
+// TestAddressPubkeyConverter represents an address public key converter
+var TestAddressPubkeyConverter, _ = pubkeyConverter.NewBech32PubkeyConverter(32)
 
-// TestPubkeyConverterBLS represents a public key converter from BLS public keys
-var TestPubkeyConverterBLS, _ = pubkeyConverter.NewHexPubkeyConverter(96)
+// TestValidatorPubkeyConverter represents an address public key converter
+var TestValidatorPubkeyConverter, _ = pubkeyConverter.NewBech32PubkeyConverter(96)
```

Comment:
normally this is a hexConverter.

Reply:
Wanted to check if the rest of the elrond-go code has the pubkeyConverter correctly injected. 
I think we should keep this in order to check future implementations not use a hardcoded hex.Encode inside a method. Maybe we decide to switch also to Bech32 form of the validator pubkey. Maybe with the `erv1` prefix :)

Reply:
ok